src/components/comment/Comment.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+class Comment extends React.Component {
+
+    constructor(props) {
+        super(props);
```

Comment:
constructor is not necessary if you are not going to configure some value here.

Reply:
I add state in the constructor.